lib/utils.js

```diff
@@ -106,7 +106,8 @@ const utils = {
           writeToken: 'GITHUB_TOKEN'
         },
         provider: 'github'
-      }
+      },
+      prependChangelog: true
```

Comment:
Looks like you called it `prependChagnelog` here, but `processChangelog` in the `README.md`